### PR TITLE
add note for cloud customers about manually enabling embeddings

### PR DESCRIPTION
**Pull request by jasonhawkharris**
## Test plan
Doc update

### CODE REVIEW

File: doc/cody/explanations/code_graph_context.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Embeddings for relevant code files must be enabled for each repository that you'
 > embeddings will be enabled for all repositories by default.
 
 Embeddings are automatically enabled and configured once [Cody is enabled](../quickstart.md). You can also [use third-party embeddings provider directly](#using-a-third-party-embeddings-provider-directly) for embeddings.
+> NOTE: `Embeddings` and `completions` are not automatically enabled for cloud customers.
+> Cloud customers must enable `embeddings` and `completions` in the site-admin panel.
```
Comment (bobheadxi) [inline doc/cody/explanations/code_graph_context.md:26]: In 5.1, `completions` will be enabled by default when the new `cody.enabled` is configured

File: doc/cody/explanations/code_graph_context.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ Embeddings for relevant code files must be enabled for each repository that you'
...
+> If these are not enabled, visiting `site-admin/cody` will result in a `404` error.
```
Comment (bobheadxi) [inline doc/cody/explanations/code_graph_context.md:27]: I think the lack of `site-admin/cody` is the key thing here, we could word the notice like this:

```suggestion

> NOTE: Unless both completions and embeddings are configured, the `/site-admin/cody` page will not be available.
```

File: doc/cody/explanations/code_graph_context.md (hunk 1 above)
Comment (jasonhawkharris) [reply doc/cody/explanations/code_graph_context.md:26]: @bobheadxi. Got it, but not embeddings?

Comment (bobheadxi) [reply doc/cody/explanations/code_graph_context.md]: I think embeddings will continue to remain off-by-default until Sourcegraph embeddings is available, but the context team might be able to share more about this (cc'd for review)

**Review by bobheadxi — APPROVED** [review]

**Review by unknwon — APPROVED** [review]
LGTM as an improvement to the status quo. 

@bobheadxi - Do you think the wording here need to be updated once the Cody Gateway is the only option for Cloud customers in 5.1?

**Comment by bobheadxi** [comment]
> Do you think the wording here need to be updated once the Cody Gateway is the only option for Cloud customers in 5.1?

Not yet - see https://github.com/sourcegraph/controller/pull/805, while we will currently be enforcing it for migration I don't think the door has been closed entirely yet on BYO accounts in Cloud, post 5.1 there might be an ask for BYO accounts.

**Pull request merged**